src/OrleansRuntime/Startup/IStartup.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Runtime.MembershipService;
+using Orleans.Runtime.ReminderService;
+
+namespace Orleans.Runtime.Startup
+{
+    public interface IStartup
```

Comment:
Mirosoft.AspNetCore.HosttingIStartup has two methods
1. Configure(Microsoft.AspNetCore.Builder.IApplicationBuilder)
2.  ConfigureServices(Microsoft.Extensions.DependencyInjection.IServiceCollection)

we definitely want to borrow method 2, but I'm not sure about method 1, since I IApplicationBuilder is pretty web server builder pattern, here is its interface doc: https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Builder/IApplicationBuilder/